workflow/common.go

```diff
@@ -0,0 +1,72 @@
+package workflow
+
+import (
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
+)
+
+var defaultVolumeName = "DefaultVolumeTest"
+var defaultInitiatorGrpName = "DefaultInitiatorgrpTest"
+var defaultVolCollName = "DefaultVolCollTest"
+
+func config() (*service.NsGroupService, error) {
+	groupService, err := service.NewNsGroupService("", "admin", "admin", "v1", true)
```

Comment:
What does it mean for the address to be empty?  Is customization required?  An alternative is to read the connection details from the environment.

Passwords should never be submitted to source repositories.